TerriaJS/terriajs
# support custom URL parameters in a GetLegendGraphic request for a layer without a style configured

### What this PR does

Fixes #7198 

This PR addresses aforementioned issue by reusing an existing constant `QUERY_PARAMETERS_TO_REMOVE` to create a base URL for a legendGraphic request. I've added a new getter on `WebMapServiceCatalogItem` so the getter can be reused in the future.

### Test me

A unit test is added to test this behaviour. The new test was failing before I applied the proposed change.

Manual testing: 

The 'before' situation can be viewed with this [ci.terria.io link](http://ci.terria.io/main/#clean&https://gist.githubusercontent.com/sidneygijzen/cb49e686747d4402d523630485ff3f28/raw/3c89e2d3453a229e4e4429ebf2f2cbe745789b14/terriajs-7198.json). If one:
- clicks 'explore map data'
- selects 'test' and then 'af250m_nutrient dataset and
- adds map layer 'Africa SoilGrids nutrients - Extractable Aluminium (Al)'

After adding the layer, the legendGraphic fails to load in the 'before' situation.

[This gist](https://gist.githubusercontent.com/sidneygijzen/cb49e686747d4402d523630485ff3f28/raw/3c89e2d3453a229e4e4429ebf2f2cbe745789b14/terriajs-7198.json) can be used in testing the 'after' situation. When adding aforementioned layer, the legend graphic should be visible. I've tried to prepare an 'after' test by using ci.terria.io, but I think I can only use this method with branches existing on the official terriajs repo? If not, I would love to know how to create a ci.terria.io link with a branch from a forked repo. 

### Checklist
